lib/rpush/client/active_model/fcm/notification.rb

```diff
@@ -0,0 +1,121 @@
+module Rpush
+  module Client
+    module ActiveModel
+      module Fcm
+        module Notification
+          FCM_PRIORITY_HIGH = Rpush::Client::ActiveModel::Apns::Notification::APNS_PRIORITY_IMMEDIATE
+          FCM_PRIORITY_NORMAL = Rpush::Client::ActiveModel::Apns::Notification::APNS_PRIORITY_CONSERVE_POWER
+          FCM_PRIORITIES = [FCM_PRIORITY_HIGH, FCM_PRIORITY_NORMAL]
+
+          ROOT_NOTIFICATION_KEYS = %w[title body image].freeze
+          ANDROID_NOTIFICATION_KEYS = %w[icon tag color click_action body_loc_key body_loc_args title_loc_key
+                                         title_loc_args channel_id ticker sticky event_time local_only
+                                         default_vibrate_timings default_light_settings vibrate_timings
+                                         visibility notification_count light_settings sound].freeze
+
+          def self.included(base)
+            base.instance_eval do
+              validates :device_token, presence: true
+              validates :priority, inclusion: { in: FCM_PRIORITIES }, allow_nil: true
+
+              validates_with Rpush::Client::ActiveModel::PayloadDataSizeValidator, limit: 4096
+              validates_with Rpush::Client::ActiveModel::RegistrationIdsCountValidator, limit: 1000
```

Comment:
[NOTE]: These are the same validations as current GCM, so we're good. I think this is good.